src/components/EditView.js

```diff
@@ -29,6 +29,7 @@ const EditView = (props) => {
   const { update, id, task } = props
   const [todo, setTodo] = useState('')
   const [done, setDone] = useState(false)
+  console.log('Id: ', id)
```

Comment:
stray console.logs? Do you want to keep these?

Reply:
Yes